Clarify an endpoint can send less ACKs when processing packets in a batch. (#77)
quicwg/ack-frequency: draft-ietf-quic-ack-frequency.md

To avoid sending multiple acknowledgements in rapid succession, an endpoint MAY
process all packets in a batch before determining whether a threshold has been
met and an acknowledgement is to be sent in response.  An endpoint MAY

> send less acknowledgements due to processing packets in a batch than
> it would if the packets were processed separately.

Reviewer: ```suggestion
wait to process all packets in a batch before sending an
acknowledgement.
```
